tools/build/ci_install_libpq.sh

```diff
@@ -18,23 +25,21 @@ case "$libpq" in
         ;;
 
     oldest)
+        echo "deb http://apt.postgresql.org/pub/repos/apt ${rel}-pgdg main 10" \
+            | sudo tee -a /etc/apt/sources.list.d/pgdg.list > /dev/null
+        setup_repo
+
         pqver=$(apt-cache show libpq5 | grep ^Version: | tail -1 | awk '{print $2}')
-        sudo apt-get -qq -y --allow-downgrades install \
-            "libpq-dev=${pqver}" "libpq5=${pqver}"
+        sudo apt-get -qq -y --allow-downgrades install "libpq-dev=${pqver}" "libpq5=${pqver}"
```

Comment:
Observation: there are `sudo` everywhere. We might as well drop all of them and call this script using `sudo` in the CI workflow?